### PR TITLE
Step monitor go live

### PR DESCRIPTION
**Pull request by bradleycamacho**
Got the go ahead to go live with step monitor changes.

### CODE REVIEW

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx [hunk 1]
```diff
@@ -70,6 +70,26 @@ These are the four types of synthetic monitors:
...
 
+    <tr>
+      <td>
+        Step monitor
+
+        API name: `STEP_MONITOR`
+      </td>
+
+      <td>
+        Step Monitors are advanced monitors which require no code to set up.
```
Comment (barbnewrelic) [inline src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx]: Recommend double-check consistency of "Step Monitor" or "Step monitor" or "step monitor" throughout

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx [hunk 2]
```diff
@@ -70,6 +70,26 @@ These are the four types of synthetic monitors:
...
+      </td>
+
+      <td>
+        Step Monitors are advanced monitors which require no code to set up. 
+
+        The monitor can be configured to:
+        - Navigate to a URL 
+        - Type text
+        - Click an element
+        - Assert text
```
Comment (barbnewrelic) [inline src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx:87]: s/be "assert" or "insert" for this bullet and next?

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx [hunk 3]
```diff
@@ -67,6 +67,31 @@ There are several **types** of synthetic monitor to add. **Ping** monitors ensur
 
     Wait a few minutes, then check your monitor from the [**Monitors** index](/docs/synthetics/new-relic-synthetics/dashboards/synthetics-monitors-dashboard-access-your-monitors).
   </Collapser>
+
+   <Collapser
+    id="stepmonitor"
+    title="Add a step monitor"
```
Comment (barbnewrelic) [inline src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx:73]: same comment as before re: capitalization

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx [hunk 4]
```diff
@@ -67,6 +67,31 @@ There are several **types** of synthetic monitor to add. **Ping** monitors ensur
...
+  >
+    1. Go to [**one.newrelic.com**](http://one.newrelic.com/) **> Synthetics > <Icon name="fe-plus-circle"/>
+       Create monitor**.
+    2. Select a Step monitor as the [monitor type](#setting-type).
```
Comment (barbnewrelic) [inline src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx:77]: same comment re: capitalization

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx [hunk 5]
```diff
@@ -67,6 +67,31 @@ There are several **types** of synthetic monitor to add. **Ping** monitors ensur
...
+    3. Specify a [name](#setting-name) and choose a [frequency](#setting-frequency) to determine how often each location will run your monitor.
+    4. Select the [locations](#setting-location) from which you want your monitor to run.
+    5. Build your monitor by selecting from the preconfigured steps at the bottom of the UI:
+        * Navigate to a URL 
+        * Type text
+        * Click an element
+        * Assert text
```
Comment (barbnewrelic) [inline src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx:84]: Same comment as before re: "assert" vs "insert"

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx [hunk 6]
```diff
@@ -67,6 +67,31 @@ There are several **types** of synthetic monitor to add. **Ping** monitors ensur
...
+        * Assert an element 
+        * Secure a credential
+
+        Use the instructions on the right side of the UI to help locate elements by CSS class, HTML ID, link text, or XPath. 
+    6. Select **Save monitor** to confirm.
+
+  Wait a few minutes, then check your monitor from the [**Monitors** index](/docs/synthetics/new-relic-synthetics/dashboards/synthetics-monitors-dashboard-access-your-monitors).
```
Comment (barbnewrelic) [inline src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx:91]: Is path and "index" still accurate in NR One UI?

**Review by barbnewrelic — APPROVED** [review]
Looks good! Please review my comments, then go ahead and merge. I know you're on a timeline. :-)

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx (hunk 1 above)
Comment (bradleycamacho) [reply src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx]: Just pushed up a commit fixing this, thanks!

File: src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx (hunk 2 above)
Comment (bradleycamacho) [reply src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx:87]: It should be "assert text" as that's the UI text

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx (hunk 6 above)
Comment (bradleycamacho) [reply src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx:91]: Yup! There are some large UI changes upcoming, but those are not rolled out yet.

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx (hunk 4 above)
Comment (bradleycamacho) [reply src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx:77]: Fixed

File: src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx (hunk 3 above)
Comment (bradleycamacho) [reply src/content/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors.mdx:73]: fixed

**Review by barbnewrelic — APPROVED** [review]
I made several comments and clicked the Approve button

**Pull request merged**